tickets/src/test/java/no/unit/nva/publication/ticket/create/CreateTicketHandlerTest.java

```diff
@@ -657,6 +658,26 @@ void userWithAccessRightManageDoiShouldNotBeAbleToAutoPublishFilesWhenAllowedToP
         assertThat(file, is(instanceOf(UnpublishedFile.class)));
     }
 
+    @Test
+    void shouldSetFinalizedByFromRequestUtilsWhenTicketIsAutoApproved()
+        throws ApiGatewayException, IOException {
+        var publication = createPersistedPublishedPublication();
+        var publicationPublisher = publication.getPublisher().getId();
+        var curatorName = randomString();
+        var requestBody = constructDto(PublishingRequestCase.class);
+        var request = createHttpTicketCreationRequestWithApprovedAccessRight(requestBody, publication, curatorName,
+                                                                             publicationPublisher,
+                                                                             MANAGE_PUBLISHING_REQUESTS, MANAGE_DOI);
+        handler.handleRequest(request, output, CONTEXT);
+        var response = GatewayResponse.fromOutputStream(output, Void.class);
+        assertThat(response.getStatusCode(), is(equalTo(HTTP_CREATED)));
+
+        var ticket = fetchTicket(response);
+
+        assertThat(ticket.getFinalizedBy().toString(), is(not(equalTo(publication.getResourceOwner().getOwner()))));
```

Comment:
Not really necessary, but added to reiterate what we do not want. Should remove?